tests/Regress.hs

```diff
@@ -41,7 +41,7 @@ regressions = [] ++
         hs @=? nub hs
 #if WORD_SIZE_IN_BITS == 64
     , testCase "64 bit Text" $ do
```

Comment:
Why this test is at all in `unordered-containers`?

Reply:
This is hashable on which unordered containers builds, 
you've added this in august 0f8d8d1df3c6671ef69a02073aed7e92a1ad9de1
I don't know what it does. (presumably, there was a regression in the hash of text?).

Reply:
Now I wonder what it was related to.

Reply:
Ah, now I remember. Well yes, I'm not "fixing" this. The test is there to ensure that we don't accidentally change the default salt, nor how `Text` is serialized - without reason.

`random-initial-seed` flag is there for testing only, and for testing of not hashable, so the test will stay unalrtered.

Reply:
I'm curious why I wouldn't want to change the default salt? It seems like a good workaround for that [DoS vulnarbility CSSyd](https://cs-syd.eu/posts/2021-09-11-json-vulnerability) wrote about.

I was intending to do this for a company as future proving (no public aeson endpoints for now, so this isn't particularly urgent).

Reply:
Because I don't consider `random-initial-seed` to be a security feature, it was added to aid testing, because many test-suites/doctests incorrectly assume the stability of `hashable` has functions, but it's still can be assumed that on a single system the `hashable` behaves deterministically.

See also https://github.com/commercialhaskell/stackage/issues/5878

Reply:
Hmm, yeah I encountered xmlbf breaking as well.
I guess many people expect unordered containers to be ordered in their tests for some reason.

So I'll probably enable the flag for the company because it'd be pretty silly we got DoS'ed in the future. I'm aware of it now after all, and I'm afraid we'll forget. Or maybe there are additional reasons to not change the default salt?

Reply:
If case if you haven't followed, but there is a ongoing work on fixing this in `aeson` in more principled way, with first steps already merged https://github.com/haskell/aeson/pull/866

So please calm down, especially as you don't have public `aeson` endpoints.

Reply:
I wasn't aware,
I think that's good enough for me to wait and see, thanks for all your hard work!